lib/igniter/util/io.ex

```diff
@@ -4,7 +4,7 @@ defmodule Igniter.Util.IO do
   @doc "Prompts the user for yes or no, repeating the prompt until a satisfactory answer is given"
   def yes?(prompt) do
     case String.trim(Mix.shell().prompt(prompt <> " [Yn]")) do
-      yes when yes in ["y", "Y", "yes", "YES"] ->
+      yes when yes in ["y", "Y", "yes", "YES", ""] ->
```

Comment:
we actually just changed this explicitly for that not be the case. I hadn't considered that the capital Y implies that it's the default, but it intentionally requires explicit acceptance.

Reply:
okay then we need to fix the case of the prompt.